### PR TITLE
Adds log file test fix and extends blacklist to allows for single files

### PR DESCRIPTION
**Pull request by Cyb3r-Jak3**
- Fixes log file test
- Extends blacklist to allow for a single file

Much cleaner of #75

### CODE REVIEW

File: html5validator/cli.py [hunk 1]
```diff
@@ -25,8 +25,11 @@ def parse_yaml(filename, starter):
         yaml_contents = yaml.safe_load(yaml_file)
     LOGGER.debug(yaml_contents)
     for item in yaml_contents.keys():
+        if item == "extra":
```
Comment (svenkreiss) [inline html5validator/cli.py:28]: please rename `extra` to `vnu`.

File: html5validator/cli.py [hunk 2]
```diff
@@ -90,8 +93,8 @@ def main():
     args, extra_args = parser.parse_known_args()
 
     if args.config is not None:
-        args = parse_yaml(args.config, args)
-        LOGGER.debug(args)
+        args, extra_args = parse_yaml(args.config, args)
+        LOGGER.debug(args, extra_args)
```
Comment (svenkreiss) [inline html5validator/cli.py:97]: Don't you need a format string as the first argument?
At this point, logging is also not configured yet. Maybe remove.

File: html5validator/cli.py [hunk 3]
```diff
@@ -90,8 +93,8 @@ def main():
     args, extra_args = parser.parse_known_args()
 
     if args.config is not None:
-        args = parse_yaml(args.config, args)
-        LOGGER.debug(args)
+        args, extra_args = parse_yaml(args.config, args)
```
Comment (svenkreiss) [inline html5validator/cli.py:96]: I didn't realize this before, but overwriting all command line arguments is really not great. A config file should serve as a default that can be overridden by command line args. A todo item for later.

**Review by svenkreiss — COMMENTED** [review]
Thanks! Much easier to look over.
I just have one request and then this is good to be merged.

File: html5validator/cli.py (hunk 3 above)
Comment (Cyb3r-Jak3) [reply html5validator/cli.py:96]: The issue with this is that most command-line arguments have a default so we would need to have a list of what the default arguments are to compare to what was passed.

File: html5validator/cli.py (hunk 2 above)
Comment (Cyb3r-Jak3) [reply html5validator/cli.py]: I removed the logging here. Don't know what you refer to with format string and the first argument in the path to the config file.

File: html5validator/cli.py (hunk 1 above)
Comment (Cyb3r-Jak3) [reply html5validator/cli.py]: Done

**Comment by svenkreiss** [comment]
Thanks!

**Pull request merged**